src/mesh/rectilineargrid.jl

```diff
@@ -35,12 +35,13 @@ vertex(g::RectilinearGrid{Dim}, ijk::Dims{Dim}) where {Dim} = Point(getindex.(g.
 
 xyz(g::RectilinearGrid) = g.xyz
 
-@generated function XYZ(g::RectilinearGrid{Dim,T}) where {Dim,T}
+XYZ(g::RectilinearGrid) = XYZ(xyz(g))
+
+@generated function XYZ(xyz::NTuple{Dim,<:AbstractVector{T}}) where {Dim,T}
```

Comment:
Should we leave this function in this file? Or do we move it to another file?

Reply:
Let's move it to utils.jl